From 61afbad6579395d0006d77db5837ad0d4e149c27 Mon Sep 17 00:00:00 2001
From: William Wu <william.wu@rock-chips.com>
Date: Mon, 26 Mar 2018 20:11:06 +0800
Subject: [PATCH] phy: rockchip-inno-usb2: fix a clang warning

This patch fix the following clang warning:
[clang] drivers/phy/rockchip/phy-rockchip-inno-usb2.c:1458:2:
warning: 3rd function call argument is an uninitialized value

Change-Id: Ia4a556da346323e5bc081ff1bbf267911fcdc645
Signed-off-by: William Wu <william.wu@rock-chips.com>
---
 drivers/phy/rockchip/phy-rockchip-inno-usb2.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/phy/rockchip/phy-rockchip-inno-usb2.c b/drivers/phy/rockchip/phy-rockchip-inno-usb2.c
index 66624b1e9b84..b944b2bee1b6 100644
--- a/drivers/phy/rockchip/phy-rockchip-inno-usb2.c
+++ b/drivers/phy/rockchip/phy-rockchip-inno-usb2.c
@@ -1436,7 +1436,7 @@ static irqreturn_t rockchip_usb2phy_id_irq(int irq, void *data)
 {
 	struct rockchip_usb2phy_port *rport = data;
 	struct rockchip_usb2phy *rphy = dev_get_drvdata(rport->phy->dev.parent);
-	bool cable_vbus_state;
+	bool cable_vbus_state = false;
 
 	if (!property_enabled(rphy, &rport->port_cfg->idfall_det_st) &&
 	    !property_enabled(rphy, &rport->port_cfg->idrise_det_st))
-- 
2.35.3

